projects/ad9361/src/tinyiiod.c

```diff
@@ -0,0 +1,225 @@
+/*
```

Comment:
it would be nice at some point to just reference https://github.com/analogdevicesinc/libtinyiiod  vs copying files over

Reply:
I think you are right, probably the initial version of tinyiiod that I use and modified, was taken from here.

Reply:
Reference to the project is added:
https://github.com/analogdevicesinc/no-OS/pull/143/commits/45bd09c8e49deb15a0e0819fa8928ded7ba8c6d9